server/reaction_hooks.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+// ReactionHasBeenAdded is invoked after the reaction has been committed to the database.
+//
+// Note that this method will be called for reactions added by plugins, including the plugin that
+// added the reaction.
+//
+// This demo implementation logs a message to the demo channel whenever a reaction is added to a post.
+func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	configuration := p.getConfiguration()
+
+	if configuration.disabled {
+		return
+	}
+
+	user, err := p.API.GetUser(reaction.UserId)
+	if err != nil {
+		p.API.LogError("failed to query user", "user_id", reaction.UserId)
```

Comment:
Please also log `err` to allow easier debugging

Reply:
@hanzei - Not currently included in `channel_hooks.go`, `message_hooks.go`, `team_hooks.go`. Want me to update those in this PR as well to be consistent? Some folks prefer _pure_ PR's, other's are OK w/ grouping minor things like that. Wasn't sure on your preference.

Reply:
Yes, logging the error should become good practise. I don't mind of you change the other files in this PR too, but that is not an requirement.

Reply:
Sounds good! Went ahead and updated anywhere I else I noticed it was missing